<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhantomPropertiesGui</class>
 <widget class="QMainWindow" name="PhantomPropertiesGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phantom Properties</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>5</y>
      <width>87</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phantom Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>15</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Field (T)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtField">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>5</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtComment">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>30</y>
      <width>336</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>15</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature (C)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtTemperature">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>755</x>
      <y>15</y>
      <width>86</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># of ROI sets</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtnROIsets">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPhantomName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPhantomProperties">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>821</width>
      <height>701</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblPhantomImage">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>450</width>
      <height>300</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1283</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Phantom_File"/>
    <addaction name="actionSave_Phantom_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Phantom_File">
   <property name="text">
    <string>Open Phantom File</string>
   </property>
  </action>
  <action name="actionSave_Phantom_File">
   <property name="text">
    <string>Save Phantom File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
